src/stories/Components/Accordion/Accordion.mdx

```diff
@@ -60,7 +71,16 @@ This allows multiple classes to share the same behavior. You can modify all acco
 }
 ```
 
-## Custom Accordion
+## New Accordion Variations
+
+<div
+  dangerouslySetInnerHTML={{
+    __html: createAlert({
+      title: 'Important!',
+      description: 'These patterns represent how to create new variations of the accordion for you project.',
```

Comment:
🤦 for you project... Should be for your project. This will need to change across all the components.